src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/Rules/NewLineUserSettingFormattingRule.cs

```diff
@@ -191,7 +197,7 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
             }
 
             // For Anonymous Object Creation Expression
-            if (previousToken.Kind() == SyntaxKind.CommaToken && previousToken.Parent.Kind() == SyntaxKind.AnonymousObjectCreationExpression)
+            if (previousToken.Kind() == SyntaxKind.CommaToken && previousToken.Parent.IsKind(SyntaxKind.AnonymousObjectCreationExpression))
```

Comment:
i would support a full pass that just changes all `Kind() ==` to `IsKind`.

Reply:
I'm intentionally avoiding this pass now so we can use it later to validate https://github.com/dotnet/roslyn-analyzers/issues/3496.